File: 6-kyu/Ball Upwards/BallUpwards.js
//Kata found at : https://www.codewars.com/kata/566be96bb3174e155300001b

function maxBall(v0) {
    var cv = v0 * 0.277777777;
    var t = 10 * cv / 9.81;
    if (t - parseInt(t, 10) > 0.5)  {return parseInt(t, 10) + 1;}
    else  {return parseInt(t, 10);}
}

File: 6-kyu/Ball Upwards/BallUpwards.py
# Kata found at : https://www.codewars.com/kata/566be96bb3174e155300001b
def max_ball(v0):
    cv = 0.27777777777 * v0
    G = 9.81
    t = 10 * cv / G
    return int(t) + 1 if t - int(t) > 0.5 else int(t)
